zerver/views/__init__.py

```diff
@@ -1199,10 +1244,12 @@ def json_refer_friend(request, user_profile, email=REQ()):
 @has_request_variables
 def json_set_muted_topics(request, user_profile,
                          muted_topics=REQ(validator=check_list(check_list(check_string, length=2)), default=[])):
+    # type: (HttpRequest, UserProfile, List[List[str]]) -> HttpResponse
     do_set_muted_topics(user_profile, muted_topics)
     return json_success()
 
 def add_push_device_token(request, user_profile, token, kind, ios_app_id=None):
+    # type: (HttpRequest, UserProfile, str, int, Optional[str,bool]) -> HttpResponse
```

Comment:
should have a space between `str` and `bool` in the annotations.  Also, I'm pretty sure `ios_app_id` is never a bool, so I think this could be just Optional[str].